common/src/main/java/tachyon/security/PlainSaslServer.java

```diff
@@ -39,7 +38,7 @@
  * 3.Write a JCA provider that registers the factory
  */
 public class PlainSaslServer implements SaslServer {
-  private String mAuthcid;
+  private String mAuthzid;
```

Comment:
Please do javadoc this member variable `mAuthzid`, as I didn't realize the difference that it is for authorization ID not for authentication ID from its name.